selftests/functional/test_teststmpdir.py

```diff
@@ -62,6 +62,7 @@ def run_and_check(self, cmd_line, expected_rc, env=None):
                          "%d:\n%s" % (cmd_line, expected_rc, result))
         return result
 
+    @unittest.skip("Temporary skip because of errors on Travis-CI")
```

Comment:
Nope. Not sure if someone digged into this, but the issue happens only in travis: https://travis-ci.org/avocado-framework/avocado/jobs/195092427#L2637-L2674